pkg/kapp/resources/resources.go

```diff
@@ -557,6 +557,8 @@ func IsResourceChangeBlockedErr(err error) bool {
 		return true
 	case strings.Contains(errMsg, "Internal error occurred: failed calling webhook"):
 		return true
+	case strings.Contains(errMsg, `cluster: admission webhook "validation.gatekeeper.sh" denied the request`):
```

Comment:
Should we make it more generic by using regex for `"validation.gatekeeper.sh"`.
Something like this `cluster: admission webhook (.+) denied the request`

Reply:
TBH, I am not really happy with either of them and I am open for suggestions.